_posts/2018-11-06-twirf4-talks-and-workshops.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "This Week in RustFest 4: Workshops and Talks Schedule"
+socialImageSrc: "/assets/posts/rome/twirf-4.png"
+authors:
+  - gandro
+---
+
+This Week in RustFest we finalized our schedule for Saturday and Sunday -- and we have are happy to showcase one of our amazing sponsors.
+
+# Last Batch of Tickets
+
+We have opened our third and last batch of tickets. So get them while they are still available -- you’ll be able to purchase your ticket via this <a href="https://ti.to/asquera-event-ug/rustfest-rome-2018/"><strong>🎟 Tickets 🎟</strong></a> link
+
+# Talks and Workshops at RustFest Rome
+
+Last week, we finally announced the long awaited talks lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peek of the schedule, which is now complete and available on our website: <a href="https://rome.rustfest.eu/schedule/"><strong>🎙️ Talk Schedule 🎙️</strong></a>
+
+RustFest would of course not be RustFest without workshops. We already announced [**🌉 RustBridge**](https://blog.rustfest.eu/rustbridge-rome) as our first workshop a few weeks back. This week we are super excited to announce the remaining workshops we are having on Sunday:
```

Comment:
```suggestion
RustFest of course would not be such without workshops. We already announced [**🌉 RustBridge**](https://blog.rustfest.eu/rustbridge-rome) as our first workshop a few weeks back. This week we are super excited to announce the remaining workshops we are having on Sunday:
```

Reply:
not too sure about that, though

Reply:
Me neither, I'll have to check the idiom :+1: